README.md

```diff
@@ -1,24 +1,43 @@
 # OVAL visualization as graph
+Understanding result in the blink of an eye
+
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Code Coverage](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/?branch=master) [![Build Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/build.png?b=master)](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/build-status/master) [![Code Intelligence Status](https://scrutinizer-ci.com/g/OpenSCAP/OVAL-visualization-as-graph/badges/code-intelligence.svg?b=master)](https://scrutinizer-ci.com/code-intelligence)
 
-Understanding result in the blink of an eye
+## Visualization of SCAP rule evaluation results
+This client generate tree graph from ARF xml report from OpenSCAP scan.
+## Prerequisites
+##### Minimal requirements
+   - **python3**
+   - lxml
+   
+##### Recommended requirements
+  * include minimal requirements
+  * inquirer
+
+### Installation
+
+```bash
+git clone https://github.com/OpenSCAP/OVAL-visualization-as-graph.git
+cd OVAL-visualization-as-graph
+
+# Install package with nice cli futures (recommended)
+python3 setup.py install ".[niceCli]"
+
+# Install package without futures (light version)
+python3 setup.py install .
+```
 
 ### Usage
 ```
-Command:
-    python3 run.py ARF-file.xml id-rule
-Example:
-    python3 run.py data/ssg-fedora-ds-arf.xml xccdf_org.ssgproject.content_rule_disable_host_auth
+arf-to-graph [-h] [--show-fail-rules] [--show-not-selected-rules]
```

Comment:
Don't copy-paste the help here, just write to run `arf-to-graph -h` to get help. That way, you don't have to worry about updating the README each time the interface gets updated.